packages/@okta/i18n/src/properties/login.properties

```diff
@@ -840,7 +840,7 @@ oie.numberchallenge.warning = Haven't received a push notification yet? Try open
 ## {0} refers to a number which is the correct answer for the number challenge
 oie.numberchallenge.instruction = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
 oie.numberchallenge.force.upgrade.title = Update Okta Verify
-oie.numberchallenge.force.upgrade = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifcations, update Okta Verify to the latest version, then try again.
+oie.numberchallenge.force.upgrade = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifications, update Okta Verify to the latest version, then try again.
```

Comment:
IIUC, UI will retrieve the error msg from this file instead of from okta-core, will this lead to any problem?
For example, we have changed the error msg in okta-core, but not here, the user will be shown the out-of-date msg?

Reply:
Yes, the UI looks at the i18n key sent by the server and does a look up in this file to show the message. The error message in okta-core should not have any effect on the widget as we dont use it at all, we only look at the key to add a message on the client side.